config/initializers/assets.rb

```diff
@@ -4,4 +4,5 @@
   miq_novnc.js
   spice-html5.js spiceHTML5/spicearraybuffer.js
   codemirror/modes/*.js codemirror/themes/*.css
+  container_topology.css
```

Comment:
Weird that this didn't make it a merge conflict in this PR (unless it was a straight cherry-pick out of this set or something)